integ-tests/src/cucumber/java/steps/dataservice/CreateEnvironmentSteps.java

```diff
@@ -14,29 +14,72 @@
  */
 package steps.dataservice;
 
+import com.amazonaws.blox.dataservicemodel.v1.model.EnvironmentId;
+import com.amazonaws.blox.dataservicemodel.v1.model.EnvironmentType;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.CreateEnvironmentRequest;
 import configuration.CucumberConfiguration;
 import cucumber.api.java8.En;
-import org.apache.commons.lang3.NotImplementedException;
+import java.util.UUID;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.test.context.ContextConfiguration;
 import steps.wrappers.DataServiceWrapper;
 
 @ContextConfiguration(classes = CucumberConfiguration.class)
 public class CreateEnvironmentSteps implements En {
 
+  private static final String ENVIRONMENT_NAME_PREFIX = "test";
+  private static final int ACCOUNT_ID_SIZE = 12;
+  private static final String ACCOUNT_ID = generateAccountId();
+  private static final String TASK_DEFINITION_ARN =
+      "arn:aws:ecs:us-east-1:" + ACCOUNT_ID + ":task-definition/sleep";
+  private static final String ROLE_ARN = "arn:aws:iam::" + ACCOUNT_ID + ":role/testRole";
+  private static final String CLUSTER_NAME_PREFIX = "cluster";
+
   @Autowired private DataServiceWrapper dataServiceWrapper;
 
   public CreateEnvironmentSteps() {
     When(
         "^I create an environment$",
         () -> {
-          throw new NotImplementedException("");
+          dataServiceWrapper.createEnvironment(createEnvironmentRequest());
         });
 
-    Then(
-        "^the created environment response is valid$",
-        () -> {
-          throw new NotImplementedException("");
+    When(
+        "^I create an environment named \"([^\"]*)\"$",
```

Comment:
In the create_environment.feature, we already remove the "named 'test'". So here in steps, should we still keep the name, or should we also remove it?

Reply:
we have steps both with the name and without the name being specified that's why we have both definitions

Reply:
I see. Makes sense!